scripts/sql/276_docer_lint.down.sql

```diff
@@ -0,0 +1,22 @@
+DELETE FROM plugin_step_variable WHERE plugin_step_id in (SELECT ps.id FROM plugin_metadata p inner JOIN plugin_step ps on ps.plugin_id=p.id WHERE p.plugin_version='1.0.0' and p.name='DOCKER LINT' and p.deleted=false and ps."index"=1 and ps.deleted=false);
+
+
+DELETE FROM plugin_step WHERE plugin_id = (SELECT id FROM plugin_metadata WHERE plugin_version='1.0.0' and name='DOCKER LINT' and deleted=false);
+
+
+DELETE FROM plugin_stage_mapping WHERE plugin_id =(SELECT id FROM plugin_metadata WHERE plugin_version='1.0.0' and name='DOCKER LINT' and deleted=false);
+
+
+DELETE FROM plugin_tag_relation WHERE plugin_id =(SELECT id FROM plugin_metadata WHERE plugin_version='1.0.0' and name='DOCKER LINT' and deleted=false);
+
+
+DELETE FROM pipeline_stage_step where ref_plugin_id in (SELECT id from plugin_metadata WHERE plugin_version='1.0.0' and name ='DOCKER LINT' and deleted=false);
```

Comment:
Down fails with below error :  
`ERROR:  update or delete on table "pipeline_stage_step" violates foreign key constraint "pipeline_stage_step_variable_pipeline_stage_step_id_fkey" on table "pipeline_stage_step_variable"
DETAIL:  Key (id)=(1) is still referenced from table "pipeline_stage_step_variable".`